access.py

```diff
@@ -0,0 +1,465 @@
+#! /usr/bin/python
+
+# Stupid python3
```

Comment:
I have written the sync script in `python3` only, so you can assume we are on python3 (unless you see other use cases).

Reply:
Yeh, that's fine, I think it should work in both now.

I didn't see a #! line when I started the integration and thought I remembered it being in python2 (which is the only python I have much experience with).